pom.xml

```diff
@@ -49,6 +53,103 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+			<dependency>
+				<groupId>com.google.cloud.trace.v1</groupId>
+				<artifactId>sink</artifactId>
+				<version>${cloud-trace-java.version}</version>
```

Comment:
The google ones are fine IMO. Nobody else but this project is suited to have an opinion on those and it is actually better that you have an opinion so that things are consistent.

The exclusion is key as well in case a user adds the dependency manually in a project generated with the bom.